README.md

```diff
@@ -97,6 +97,7 @@ For other CUDA versions, consider installing from source or using our
 [build script](https://github.com/facebookresearch/theseus/blob/main/build_scripts/build_wheel.sh).
 
 #### **From source**
+The simplest way to install Theseus from source is by running
```

Comment:
could also add:

... from source is by running (see below to also include BaSpaCho)